frontend/mr-admin-flate/src/components/avtaler/opprett/OpprettAvtaleContainer.tsx

```diff
@@ -145,14 +167,14 @@ function ReactHookFormContainer() {
           />
         </FormGroup>
         <FormGroup cols={2}>
-          <Select label={"Avtaleansvarlig"} {...register("avtaleansvarlig")}>
-            {isLoading && !ansatt ? (
-              <option value={""}>Laster...</option>
-            ) : (
-              <option
-                value={ansatt?.ident ?? "ukjent"}
-              >{`${navn} - ${ansatt?.ident}`}</option>
-            )}
+          <Select
+            error={errors.avtaleansvarlig?.message}
+            label={"Avtaleansvarlig"}
+            {...register("avtaleansvarlig")}
+          >
+            <option
+              value={ansatt.ident ?? ""}
+            >{`${navn} - ${ansatt?.ident}`}</option>
```

Comment:
Jeg kom til å tenke på at disse feltene der vi har preutfylte felter som strengt tatt ikke kan endres enda, burde være read-only så det blir enda tydeligere at man ikke kan gjøre endringer på dem. Sjekk ut https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly

Reply:
Bare glem kommentaren over. Jeg har tenkt at det var et input-felt og ikke select. Den er jo uansett prevalgt av defaultValues så all is good!